### PR TITLE
Feature/979 patch v2 entities eid

### PR DESCRIPTION
**Pull request by kzangeli**
Implements #979 

Added request "PATCH /v2/entities/{entity::id}

### CODE REVIEW

File: test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Intent to patch a non-existing entity - error, of course
+# 02. Create E1/attr1+attr2
+# 03. GET /v2/entities/E1
+# 04. PATCH /v2/entities/E1 attr1=4
+# 05. GET /v2/entities/E1 - see attr1=4
```
Comment (fgalan) [inline test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test]: Add step: GET /v2/entities/E1 - see attr2 in order to check that the PATCH in step 4 hasn't modified the other attribute.

Comment (kzangeli) [inline test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test]: Not sure I understand.
Step 05 makes sure that attr1 == 4.0  AND attr2 == 2.0 (untouched from step 02).

You mean to add a comment, like:

    05. GET /v2/entities/E1 - see attr1=4 AND attr2=2    ?

Comment (kzangeli) [inline test/functionalTest/cases/0979_patch_v2_entities_eid/patch_v2_entities_eid.test]: Fixed in 0eb8f78

**Comment by fgalan** [comment]
LGTM

**Pull request merged**